tests/e2e/lib.sh

```diff
@@ -665,6 +665,83 @@ check_for_stackrox_OOMs() {
     fi
 }
 
+declare -A POD_CONTAINERS_MAP
+fill_pod_containers_map() {
+    POD_CONTAINERS_MAP=()
+
+    POD_CONTAINERS_MAP["pod: central - container: central"]="central-[A-Za-z0-9]+-[A-Za-z0-9]+-central-previous.log"
+    POD_CONTAINERS_MAP["pod: central-db - container: init-db"]="central-db-[A-Za-z0-9]+-[A-Za-z0-9]+-init-db-previous.log"
+    POD_CONTAINERS_MAP["pod: central-db - container: central-db"]="central-db-[A-Za-z0-9]+-[A-Za-z0-9]+-central-db-previous.log"
+    POD_CONTAINERS_MAP["pod: config-controller - container: manager"]="config-controller-[A-Za-z0-9]+-[A-Za-z0-9]+-manager-previous.log"
+
+    POD_CONTAINERS_MAP["pod: scanner-v4 - container: matcher"]="scanner-v4-[A-Za-z0-9]+-[A-Za-z0-9]+-matcher-previous.log"
+    POD_CONTAINERS_MAP["pod: scanner-v4 - container: indexer"]="scanner-v4-[A-Za-z0-9]+-[A-Za-z0-9]+-indexer-previous.log"
+    POD_CONTAINERS_MAP["pod: scanner-v4-db - container: init-db"]="scanner-v4-[A-Za-z0-9]+-[A-Za-z0-9]+-init-db-previous.log"
+    POD_CONTAINERS_MAP["pod: scanner-v4-db - container: db"]="scanner-v4-[A-Za-z0-9]+-[A-Za-z0-9]+-db-previous.log"
+
+    POD_CONTAINERS_MAP["pod: sensor - container: sensor"]="sensor-[A-Za-z0-9]+-[A-Za-z0-9]+-sensor-previous.log"
+    POD_CONTAINERS_MAP["pod: admission-control - container: admission-control"]="admission-control-[A-Za-z0-9]+-[A-Za-z0-9]+-admission-control-previous.log"
+
+    POD_CONTAINERS_MAP["pod: collector - container: collector"]="collector-[A-Za-z0-9]+-collector-previous.log"
+    POD_CONTAINERS_MAP["pod: collector - container: compliance"]="collector-[A-Za-z0-9]+-compliance-previous.log"
+    POD_CONTAINERS_MAP["pod: collector - container: node-inventory"]="collector-[A-Za-z0-9]+-node-inventory-previous.log"
+}
+
+junit_report_pod_restarts() {
+    local check_out="${1:-}"
+
+    local previous_logs=()
+    while IFS='' read -r line; do previous_logs+=("$line"); done < <(echo "${check_out}" | grep "copied to Artifacts" | cut -d" " -f1 | sort -u)
```

Comment:
Please add a comment in the place where this log line is emitted saying that it should not be changed as there are scripts that rely on it.

Reply:
Comment is already there: https://github.com/stackrox/stackrox/blob/7e49062da60fbe153d811e42dbcedf8df10bef5a/scripts/ci/logcheck/check-restart-logs.sh#L51